Engine/src/Engine/EngineUI/GamePanel.h

```diff
@@ -7,9 +7,15 @@ namespace Engine
 	class GamePanel : public UserInterface
 	{
 	public:
-		void setInitialPosition();
+
+		virtual ~GamePanel() {}
+
 		void show(Window& window);
+
 		glm::vec2 getPosition() { return m_position; };
+
+		const int titleBarHeight = 36;
```

Comment:
const should be following the naming convention of VARIABLE_NAME

Reply:
fixed name